src/common/guac_surface.c

```diff
@@ -665,19 +909,20 @@ guac_common_surface* guac_common_surface_alloc(guac_client* client,
         guac_socket* socket, const guac_layer* layer, int w, int h) {
 
     /* Init surface */
-    guac_common_surface* surface = malloc(sizeof(guac_common_surface));
+    guac_common_surface* surface = calloc(1, sizeof(guac_common_surface));
```

Comment:
FYI, also from that man page:

> ### CONFORMING TO
> 
> C89, C99.
> 

Our code is written conforming to C99 and POSIX, so we're good here.